exo-sync/bin/setup

```diff
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-npm install --loglevel error --depth 0
+yarn install
```

Comment:
As agreed on the check-in, let's not install yarn here, but show an error message and abort.